pkg/model/replica_entity.go

```diff
@@ -40,14 +40,17 @@ type ReplicaEntity interface {
 }
 
 type replicaEntity struct {
-	env                  simulator.Environment
-	number               int
-	kubernetesClient     kubernetes.Interface
-	endpointsInformer    informers.EndpointsInformer
-	endpointAddress      corev1.EndpointAddress
-	requestsProcessing   RequestsProcessingStock
-	requestsComplete     simulator.SinkStock
-	numRequestsSinceStat int32
+	env                   simulator.Environment
+	number                int
+	kubernetesClient      kubernetes.Interface
+	endpointsInformer     informers.EndpointsInformer
+	endpointAddress       corev1.EndpointAddress
+	requestsProcessing    RequestsProcessingStock
+	requestsComplete      simulator.SinkStock
+	requestsFailed        simulator.SinkStock
+	numRequestsSinceStat  int32
+	totalCPUCapacity      int
```

Comment:
How about `totalCpuCapacityMillis` so we know the units (since it's a primitive type).

Reply:
Renamed it. See  e91267565b8ba92593c89fe969f1d65a35e99baf commit.